.github/workflows/build.yml

```diff
@@ -130,23 +130,29 @@ jobs:
       - run: openssl genrsa -out ./config/jwt.key 2048 && chmod 0777 ./config/jwt.key
       - run: npm run swagger:validate
       - run: npm run coverage-ci # Separate command to limit the number of workers to prevent timeouts
-      - name: "Post Migration test-suite"
-        if: ${{ matrix.typeorm-connection == 'mariadb' }}
-        run: npm run test-ci-migrate
       - run: git config --global --add safe.directory $GITHUB_WORKSPACE # To avoid dubious ownership
         if: ${{ matrix.typeorm-connection == 'mariadb' }}
 
       - name: "Cannot commit code coverage cross-fork"
         if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork && matrix.typeorm-connection == 'mariadb'
         run: |
             echo "::warning Cannot comment code coverage cross-fork"
+      - name: "Fetch target branch coverage from artifacts"
+        id: fetch-coverage-base
+        uses: dawidd6/action-download-artifact@v6
```

Comment:
Since the step above and below have an `if` perhaps this one should have it too?

```diff
      - name: "Fetch target branch coverage from artifacts"
        id: fetch-coverage-base
+       if: github.event_name == 'pull_request' && !github.event.pull_request.head.repo.fork && matrix.typeorm-connection == 'mariadb'
        uses: dawidd6/action-download-artifact@v
```